networks/p2p/server.go

```diff
@@ -56,15 +56,20 @@ const (
 	frameWriteTimeout = 20 * time.Second
 )
 
-// TODO-Klaytn-Node Below constants are duplicated with node packages. Use it temporarily until fix cycle import problem.
+type ConnType int
+
 const (
-	CONSENSUSNODE = iota
+	ConnTypeUndefined ConnType = iota
+	CONSENSUSNODE
 	ENDPOINTNODE
 	PROXYNODE
 	BOOTNODE
 	UNKNOWNNODE // For error case
 )
```

Comment:
Does it cause a cycle issue if the variables are placed in `node` package?

Reply:
As these are declared in `p2p` package, it does not seem to make import cycle.

Reply:
It seems okay if it can avoid import cycle. Those variable names seem to belong to `node` package, though.